packages/react/src/utils/ast-utils.spec.ts

```diff
@@ -393,7 +393,78 @@ const store = createStore(combineReducer({}));
   });
 });
 
-describe('getComponentName', () => {
+describe('findExportDeclarationsForJsx', () => {
+  [
+    {
+      testName: 'exporting a function',
+      src: `export function Test(props: TestProps) {
+        return (
+          <div>
+            <h1>Welcome to test component, {props.name}</h1>
+          </div>
+        );`,
+      expectedName: 'Test',
+    },
+    {
+      testName: 'exporting an arrow function',
```

Comment:
This may not matter, but there is also  functional component using `React.FC`.

```
import * as React from 'react';
const Test: React.FC<TestProps> =  (props) => {
  // ...
};
```